### DISCUSSION TITLE
[과목 3] 1장 SQL 수행 구조 커스텀 문제

### DISCUSSION THREAD

**Discussion by beolggoolosori** [flataex/sqlp-study#31] [Custom 연습문제]
**문제1: 오라클 데이터베이스에서 SGA(시스템 글로벌 영역)에 포함되는 것은 무엇?**

1.데이터 파일
2.리두 로그 파일
3.공유 풀
4.테이블스페이스

**문제2: 데이터베이스에서 Connection Pooling 기법이 필요한 이유는 무엇?**

1.데이터베이스의 동시 접속자 수를 제한하기 위해
2.데이터베이스 연결을 재사용하여 성능을 향상시키기 위해
3.데이터베이스에서 데이터를 더 빨리 읽기 위해
4.데이터베이스에 저장된 데이터의 무결성을 보장하기 위해

**문제3: Sequential I/O와 Random I/O의 차이점은 무엇?**

1.Sequential I/O는 여러 위치에서 데이터를 읽고 쓰는 반면, Random I/O는 연속적인 위치에서 데이터를 읽고 쓴다.
2.Sequential I/O는 한 번에 하나의 블록만 처리하는 반면, Random I/O는 한 번에 여러 블록을 처리한다.
3.Sequential I/O는 데이터를 디스크의 연속적인 위치에서 차례로 읽거나 쓰는 방식이며, Random I/O는 데이터가 디스크의 여러 위치에 흩어져 있을 때 데이터를 읽거나 쓴다.
4.Sequential I/O는 랜덤으로 데이터를 읽고 쓰는 반면, Random I/O는 순차적으로 데이터를 읽고 쓴다.

**Comment by Psjepark** [comment]
3
2
3
